app/multianalysis/a11y.js

```diff
@@ -20,6 +23,38 @@ module.exports = {
                     return metric.rule.code === 'AX_TEXT_01';
                 });
 
+                var ariaRoleNotScoped = axsResult.filter(function (metric) {
```

Comment:
nie robmy filter za kazdym razem, zrob array tych kodow [AX1,AX2...] i przefiltruj nim, bo takto to za kazdym razem robisz filtrowanie a zawsze bierzemy tylko jeden element